scripts/sync-down-from-production.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env sh
+set -xe
+
+SCRIPT_DIR=$(dirname "$0")
+
+# Load up environment variables for the local (docker) postgres
+. "$SCRIPT_DIR"/../.env
+
+# Check wheter `docker compose` or `docker-compose` is available
+if command -v docker-compose > /dev/null; then
+  DOCKER_COMPOSE=docker-compose
+elif command -v docker compose > /dev/null; then
+  DOCKER_COMPOSE="docker compose"
+else
+  echo "Neither docker compose nor docker-compose is available"
+  exit 1
+fi
+
+REMOTE_PG="${PRODUCTION_PG_URL_FOR_USER_GITHUB_ACTIONS}"
+LOCAL_PG="postgres://${PG_USERNAME}:${PG_PASSWORD}@postgres/${PG_DATABASE}"
+
+# run postgres utils inside a container so that we don't have to install them on the host
+pg_exec() {
+  (cd "${SCRIPT_DIR}/.." && ${DOCKER_COMPOSE} exec -T postgres bash -c "${*}")
+}
+
+# truncate tables users and teams and cascade
+pg_exec "psql --command='TRUNCATE TABLE users, teams CASCADE;' ${LOCAL_PG}"
+# insert users from production
+pg_exec "psql --command=\"\\COPY (SELECT * FROM users) TO STDOUT (FORMAT CSV, DELIMITER ';')\" \"${REMOTE_PG}\" | psql --command=\"\\COPY users FROM STDIN (FORMAT CSV, DELIMITER ';')\" \"${LOCAL_PG}\""
+# insert teams from production
+pg_exec "psql --command=\"\\COPY (SELECT * FROM teams) TO STDOUT (FORMAT CSV, DELIMITER ';')\" \"${REMOTE_PG}\" | psql --command=\"\\COPY teams FROM STDIN (FORMAT CSV, DELIMITER ';')\" \"${LOCAL_PG}\""
```

Comment:
rather than `select * from teams` can we please omit column `submission_email` from data syncs? basically, we don't want to accidently send live council contacts emails when we're testing from pizzas/non-prod envs! more context over in #1400